sentry_sdk/integrations/stdlib.py

```diff
@@ -123,14 +127,18 @@ def _init_argument(args, kwargs, name, position, setdefault_callback=None):
 
     if name in kwargs:
         rv = kwargs[name]
-        if rv is None and setdefault_callback is not None:
-            rv = kwargs[name] = setdefault_callback()
+        if setdefault_callback is not None:
```

Comment:
I’m probably lacking context, but shouldn’t this also retain the check for None?

Reply:
Checking what for `None`?

Reply:
Oh wait, just realizing that you’re now passing rv into that callback.